### DISCUSSION TITLE
Method to Un-register account

### DISCUSSION THREAD

**Discussion by hmendez-figure** [WalletConnect/WalletConnectFlutterV2#249] [Ideas]
**Is your feature request related to a problem? Please describe.**
Since `registerAccount` is used to add internal accounts for the  construction of `Namespaces` on session proposals, should there be a method to un-register an account? I don't see way to do remove accounts.

**Describe the solution you'd like**
If it makes sense, a method should be added to un-register account.

**Comment by quetool** [comment]
Hello @hmendez-figure! Could you elaborate a bit more?
